.env.qa

```diff
@@ -15,6 +15,7 @@ DISCORD_POAP_MANAGERS=159014522542096384,879063616895197224,357230554158202890,3
 NOTION_TOKEN=
 NOTION_DEV_GUILD_PROJECTS_DATABASE_ID=7299abc920fd45b2be61fc74216fea0d
 NOTION_GUEST_PASS_DATABASE_ID=30087b23eab34e93a200ed3603eb5e95
+NOTION_SCOAP_SQUAD_DB_ID=d184e1555e6c489f812b5069fdfec4b2
```

Comment:
We need to create this page in the DAO Notion so it can be added to `.env.prod`, as well as the Discord channels in the DAO server. I'll talk to Above Average Joe about placement. Doubtful it will happen over the weekend, as everyone seems to take the weekend off, but we can update as soon as these pages and channels are set up.

Reply:
Ok, please note that the field names of the notion table should match those specified in service/constants/constants.ts SCOAP_SQUAD_NOTION_FIELDS under field_name. If anybody wants to change field names for some reason they can be updated in this constant too.